ui/src/testing/stubs/user-admin-api-stub.ts

```diff
@@ -0,0 +1,21 @@
+import { AccessBypassRequest, AdminUserListResponse, EmptyResponse, Profile, UserAdminApi } from 'generated/fetch';
+import { ProfileStubVariables } from './profile-api-stub';
+import { stubNotImplementedError } from './stub-utils';
+
+export class UserAdminApiStub extends UserAdminApi {
+  constructor(public profile = ProfileStubVariables.PROFILE_STUB) {
+    super(undefined, undefined, (..._: any[]) => { throw stubNotImplementedError; });
```

Comment:
This incantation provides a fallback implementation for any method we don't override in the service, and makes it throw this stub exception so that test fails